src/SFA.DAS.RoatpGateway.Domain/Roatp/OrganisationType.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+namespace SFA.DAS.RoatpGateway.Domain.Roatp
+{
+    public class OrganisationType
+    {
+        public int Id { get; set; }
+        public string Type { get; set; }
+        public string Description { get; set; }
+        public const int Unassigned = 0;
```

Comment:
What does Unassigned do? Looks a bit odd in a POCO class

Reply:
It was copied from Admin Service, where OrganisationType.Unassigned is used in logic, it's not required for Gateway so have removed